routers/users.py

```
@@ -0,0 +1,369 @@
+    # --- 3. 데이터 저장 ---
+    # ID 생성
+    new_id = 1
+    if users:
+        new_id = max(u["id"] for u in users) + 1
+
+    # 이미지 파일 저장
+    image_url = None
+    if profileImage:
+        # 파일명 안전하게 생성
+        ext = profileImage.filename.split(".")[-1]
+        safe_filename = f"{new_id}_{int(datetime.now().timestamp())}.{ext}"
```

Ruff (0.14.13)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)